src/app/project-overview/component.ts

```diff
@@ -75,6 +84,43 @@ export class ProjectOverviewComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit(): void {
+    this._initSubscriptions();
+    this._loadData();
+  }
+
+  ngOnDestroy(): void {
+    this._unsubscribe.next();
+    this._unsubscribe.complete();
+    this._unsubscribeLoadMembers.complete();
+  }
+
+  hideFirstVisitToOverviewPageMessage(): void {
+    this.firstVisitToOverviewPage = this._cookieService.get(this._cookieName);
+  }
+
+  hasPermission(view: View): boolean {
```

Comment:
```suggestion
  hasViewPermission(view: View): boolean {
```

Since we're only checking permission to `View`